k8s/utils.go

```diff
@@ -0,0 +1,48 @@
+package k8s
```

Comment:
Try to avoid names like `utils.go` in Go. Read about it more [here](https://dave.cheney.net/2019/01/08/avoid-package-names-like-base-util-or-common)

Reply:
you can just call it `k8s/client.go`.